dietpi/dietpi-banner

```diff
@@ -56,6 +56,11 @@
 	#/////////////////////////////////////////////////////////////////////////////////////
 	# Main Loop
 	#/////////////////////////////////////////////////////////////////////////////////////
+	export TERM=${TERM:-dumb}
+	if (( "$TERM" == "dumb" )); then
```

Comment:
Yes, exit is triggered on *scp* with openssh server and **not** triggered on a *ssh* login (just it should be). Thanks!